### ISSUE TITLE
Network plot for top-N features

### ISSUE THREAD

**Issue by AsyaOrlova** [mmschlk/shapiq#240]
Is it possible to create a network plot only for top-N features in the dataset? When there are a lot of features (ex. 200), the plot starts being noisy. Thank you!

**Comment by mmschlk** [comment]
Yes this should be possible! However, @AsyaOrlova, could you be a bit more specific what should be selected and on what grounds?As I understand it, you would like to select a lower number of features to put onto the circle, right? Would it be a solution to create a „test“ feature that will include all other interactions? Or would you rather remove all features that make up only a fraction (let’s say < 5%) of all the interactions?

Thanks for your interest! 😊

**Comment by mmschlk** [comment]
We added the [`InteractionValues.get_subset(players: list[int]) -> InteractionValues`](https://github.com/mmschlk/shapiq/blob/2fd4b43608f1bc6a0060f52ea2a0bd399e958902/shapiq/interaction_values.py#L520C5-L566C1) function to create at least subsets of players/features which can then be used to plot. Closing this for the time being as it is unclear how to proceed with this further at the moment. :)
